### PR TITLE
Cu 8696pcytb create settings patryk kosiski

### PR DESCRIPTION
**Pull request by JimTheCat**
Add new root for settings,
First version of settings page,
Update icon library.

Refs: CU-8696pcytb

### CODE REVIEW

**Comment by JimTheCat** [comment]
Task linked: [CU-8696pcytb Create settings](https://app.clickup.com/t/8696pcytb)

**Comment by JimTheCat** [comment]
This is how settings tab looks like:
image

File: frontend/src/Pages/Settings/Menu/Privacy/Privacy.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+export const Privacy = () => {
+  return (
+    <Box>
+
+      <Fieldset legend="Overall">
+        {/* Privacy settings */}
+        <NativeSelect label={"Who can see your profile?"} data={["Everyone", "Only friends", "Nobody"]}/>
+        <NativeSelect label={"Who can see your posts?"} data={["Everyone", "Only friends", "Nobody"]} mt={"md"}/>
+        <NativeSelect label={"Who can see your friends?"} data={["Everyone", "Only friends", "Nobody"]} mt={"md"}/>
+
```
Comment (KinTrae) [inline frontend/src/Pages/Settings/Menu/Privacy/Privacy.tsx:12]: additional enter

File: frontend/src/Pages/Settings/Menu/Privacy/Privacy.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      </Fieldset>
+
```
Comment (KinTrae) [inline frontend/src/Pages/Settings/Menu/Privacy/Privacy.tsx:14]: same goes here

**Review by KinTrae — APPROVED** [review]
awesome!

**Pull request merged**